.circleci/config.yml

```diff
@@ -116,6 +124,10 @@ jobs:
                 submit:true \
                 api_token:$HOCKEY_APP_TOKEN \
                 app_id:$HOCKEY_APP_IOS_ALPHA_APP_ID \
+                fabric_api_token:$FABRIC_API_KEY \
+                fabric_build_secret:$FABRIC_BUILD_SECRET \
+                fabric_groups:$FABRIC_GROUPS \
+                ci_build_num:$CIRCLE_BUILD_NUM \
```

Comment:
Fabric ignores new builds with the same build number so for internal builds we'll just use the CI build number. On Android this is done through the gradle.properties file.